src/daemon/rpc_command_executor.cpp

```diff
@@ -2246,4 +2247,42 @@ bool t_rpc_command_executor::check_blockchain_pruning()
     return true;
 }
 
+bool t_rpc_command_executor::rpc_payments()
+{
+    cryptonote::COMMAND_RPC_ACCESS_DATA::request req;
+    cryptonote::COMMAND_RPC_ACCESS_DATA::response res;
+    std::string fail_message = "Unsuccessful";
+    epee::json_rpc::error error_resp;
+
+    if (m_is_rpc)
+    {
+        if (!m_rpc_client->json_rpc_request(req, res, "rpc_access_data", fail_message.c_str()))
+        {
+            return true;
+        }
+    }
+    else
+    {
+        if (!m_rpc_server->on_rpc_access_data(req, res, error_resp) || res.status != CORE_RPC_STATUS_OK)
+        {
+            tools::fail_msg_writer() << make_error(fail_message, res.status);
+            return true;
+        }
+    }
+
+    const uint64_t now = (boost::posix_time::microsec_clock::universal_time() - cryptonote::rpc_payment_epoch).total_seconds();
+    uint64_t balance = 0;
+    for (const auto &entry: res.entries)
+    {
+      tools::msg_writer() << entry.client << "  " << entry.balance << "/" << entry.credits_total
```

Comment:
Maybe a little header or help text line would be nice, to make clear the order of the info, something like `client id, balance, total credits, (nonces good / stale / bad / dupe)`